### PR TITLE
add MESMER-M notebook

### PR DESCRIPTION
**Pull request by snath-xoc**
- [ ] branch to eventually merge mesmer-m 
 - [ ] Tests need to be added

### CODE REVIEW

File: mesmer_m/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#MESMER-M code
```
Comment (mathause) [inline mesmer_m/README.md:1]:
```suggestion
# MESMER-M code

```

**Comment by mathause** [comment]
@snath-xoc let me know if you need support with this (however, I'll be on holidays next week). It could also be a good idea to save the notebook as a markdown (`*.md`) or python (`*.py`) file before committing it (depending on how it looks).

Also the following might be interesting: https://medium.com/capital-fund-management/automated-reports-with-jupyter-notebooks-using-jupytext-and-papermill-619e60c37330 (cc @znicholls)

**Comment by snath-xoc** [comment]
> @snath-xoc let me know if you need support with this (however, I'll be on holidays next week). It could also be a good idea to save the notebook as a markdown (`*.md`) or python (`*.py`) file before committing it (depending on how it looks).
> Also the following might be interesting: https://medium.com/capital-fund-management/automated-reports-with-jupyter-notebooks-using-jupytext-and-papermill-619e60c37330 (cc @znicholls)

Sorry for the late reply, I was coincidentally on holiday haha. I will get started on uploading the notebook soon (latest next week). Thank you for the link will take a look!

**Review by mathause — APPROVED** [review]
I'll approve this as is. The notebook forms the basis of MESMER-M that will be subsequently refactored.

**Pull request merged**